pg_search/src/postgres/customscan/aggregatescan/privdat.rs

```diff
@@ -116,8 +115,8 @@ impl AggregateType {
             let const_node = ConstNode::try_from(args.get_ptr(1)?)?;
             let missing = match TantivyValue::try_from(const_node) {
                 // return None and bail if the conversion is lossy
-                Ok(TantivyValue(OwnedValue::U64(missing))) => f64::from_u64(missing),
-                Ok(TantivyValue(OwnedValue::I64(missing))) => f64::from_i64(missing),
+                Ok(TantivyValue(OwnedValue::U64(missing))) => f64_exact_from(missing as f64),
+                Ok(TantivyValue(OwnedValue::I64(missing))) => f64_exact_from(missing as f64),
```

Comment:
Currently the conversion is happening before the call to the method.
```suggestion
                Ok(TantivyValue(OwnedValue::U64(missing))) => f64_exact_from(missing),
                Ok(TantivyValue(OwnedValue::I64(missing))) => f64_exact_from(missing),
```